crates/uv/tests/it/lock_conflict.rs

```diff
@@ -5281,65 +5281,65 @@ fn extra_inferences() -> Result<()> {
         version = "2.5.0"
         source = { registry = "https://pypi.org/simple" }
         dependencies = [
-            { name = "alembic" },
-            { name = "apache-airflow-providers-common-sql", version = "1.8.1", source = { registry = "https://pypi.org/simple" } },
-            { name = "apache-airflow-providers-ftp", version = "3.6.1", source = { registry = "https://pypi.org/simple" } },
-            { name = "apache-airflow-providers-http", version = "4.7.0", source = { registry = "https://pypi.org/simple" } },
-            { name = "apache-airflow-providers-imap", version = "3.4.0", source = { registry = "https://pypi.org/simple" } },
-            { name = "apache-airflow-providers-sqlite", version = "3.5.0", source = { registry = "https://pypi.org/simple" } },
-            { name = "argcomplete" },
-            { name = "attrs" },
-            { name = "blinker" },
-            { name = "cattrs" },
-            { name = "colorlog" },
-            { name = "configupdater" },
-            { name = "connexion", extra = ["flask", "swagger-ui"] },
-            { name = "cron-descriptor" },
-            { name = "croniter" },
-            { name = "cryptography" },
-            { name = "deprecated" },
-            { name = "dill" },
-            { name = "flask" },
-            { name = "flask-appbuilder", version = "4.1.4", source = { registry = "https://pypi.org/simple" } },
-            { name = "flask-caching" },
-            { name = "flask-login" },
-            { name = "flask-session" },
-            { name = "flask-wtf" },
-            { name = "graphviz" },
-            { name = "gunicorn" },
-            { name = "httpx" },
-            { name = "itsdangerous" },
-            { name = "jinja2" },
-            { name = "jsonschema" },
-            { name = "lazy-object-proxy" },
-            { name = "linkify-it-py" },
-            { name = "lockfile" },
-            { name = "markdown" },
-            { name = "markdown-it-py" },
-            { name = "markupsafe" },
-            { name = "marshmallow-oneofschema" },
-            { name = "mdit-py-plugins" },
-            { name = "packaging" },
-            { name = "pathspec" },
-            { name = "pendulum" },
-            { name = "pluggy" },
-            { name = "psutil" },
-            { name = "pygments" },
-            { name = "pyjwt" },
-            { name = "python-daemon" },
-            { name = "python-dateutil" },
-            { name = "python-nvd3" },
-            { name = "python-slugify" },
-            { name = "rich" },
-            { name = "setproctitle" },
-            { name = "sqlalchemy" },
-            { name = "sqlalchemy-jsonfield" },
-            { name = "tabulate" },
-            { name = "tenacity" },
-            { name = "termcolor" },
-            { name = "typing-extensions" },
-            { name = "unicodecsv" },
-            { name = "werkzeug" },
+            { name = "alembic", marker = "extra == 'extra-3-pkg-x1'" },
```

Comment:
I'm having some trouble figuring out why the `2.6.0` version of Airflow doesn't have these on it... I guess it's because that's the "default" version that gets installed if no extras are enabled?

Reply:
Yeah I think so. There is a non-optional dependency on `quickpath-airflow-operator`, and the lock entry for that package is this:

```toml
[[package]]
name = "quickpath-airflow-operator"
version = "1.0.2"
source = { registry = "https://pypi.org/simple" }
dependencies = [
    { name = "apache-airflow", version = "2.5.0", source = { registry = "https://pypi.org/simple" }, marker = "extra == 'extra-3-pkg-x1'" },
    { name = "apache-airflow", version = "2.6.0", source = { registry = "https://pypi.org/simple" }, marker = "extra == 'extra-3-pkg-x2' or extra != 'extra-3-pkg-x1'" },
]
```

I think that conflict marker on `2.6.0` could be further simplified to `extra != 'extra-3-pkg-x1'`. (A good example of how conflict marker simplification is not perfect.)